WhaleSpotting/Controllers/AdminController.cs

```diff
@@ -25,4 +25,23 @@ public IActionResult Index()
 
         return View(unapprovedSightings);
     }
+     [HttpPost]
+        public IActionResult ApproveSightings(List<int> selectedSightingIds)
+        {
+            if (selectedSightingIds != null && selectedSightingIds.Any())
+            {
+                var selectedSightings = _context.Sightings!
+                    .Where(s => selectedSightingIds.Contains(s.Id))
+                    .ToList();
+
+                foreach (var sighting in selectedSightings)
+                {
+                    sighting.Approved = true;
+                }
+
+                _context.SaveChanges();
+            }
+
+            return RedirectToAction("Index");
+        }
```

Comment:
Logic here all looks good, but the indenting looks slightly out:
```suggestion
    [HttpPost]
    public IActionResult ApproveSightings(List<int> selectedSightingIds)
    {
        if (selectedSightingIds != null && selectedSightingIds.Any())
        {
            var selectedSightings = _context.Sightings!
                .Where(s => selectedSightingIds.Contains(s.Id))
                .ToList();

            foreach (var sighting in selectedSightings)
            {
                sighting.Approved = true;
            }

            _context.SaveChanges();
        }

        return RedirectToAction("Index");
    }
```

It would also be nice to add a line break between this function and the one before!

Reply:
Formatting updated